<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_employes</class>
 <widget class="QMainWindow" name="gestion_employes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_employes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1120</width>
      <height>650</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(88, 184, 167);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pics/BG/Rectangle 69.png</pixmap>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>240</width>
      <height>550</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>261</width>
        <height>591</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/pics/BG/Rectangle 70.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>employes_id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="employe_id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="age">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="salaire">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heures_de_travail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>salaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>heure de travail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>genre</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="homme">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>HOMME</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="femme">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>FEMME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(17, 178, 149);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(17, 178, 149);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(17, 178, 149);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>261</width>
        <height>591</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/pics/BG/Rectangle 70.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Ajouter/Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_e">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
      <property name="maxLength">
       <number>9</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_e">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>mot de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mot_de_passe_e">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_e">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>numero de telephone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_e_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>date d'embauche</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(17, 178, 149);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="rendement">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(7, 125, 73);
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Rendement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generer_pdf">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(7, 125, 73);
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Generer PDF</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/pics/BG/Rectangle 69.png</pixmap>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>681</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_e_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>451</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_e_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page1">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>380</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);

font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(9, 141, 97);</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(11, 107, 67);</string>
      </property>
      <property name="text">
       <string>employes_id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supprimer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 125, 105);

border: 1px solid #107D69;
    width: 200px;
</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>791</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/pics/BG/Rectangle 69.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pred_page2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
